app/src/main/res/layout/custom_barcode_scanner.xml

```diff
@@ -2,6 +2,15 @@
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
+    <com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
+        android:id="@+id/fabOtherOptions"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="end|bottom"
+        app:icon="@drawable/ic_baseline_unfold_more_24"
```

Comment:
I think it looks better if you just remove the icon. Unless you can find something that really fits. idk.

```suggestion
```

Reply:
It doesn't really feel better to me, the button feels empty to me. But this is very personal taste, I don't think there's an objective better way.

Reply:
Yeah. To me the current icon suggest the UI element itself will expand. So it feels off when it opens a dialog instead. But I get your PoV too. I wanted to give you my PoV. And I'll let you know if I ever think of a more appropriate icon. But it's up to you to use whatever you feel is best.